internal/pkg/cli/svc_logs.go

```diff
@@ -119,14 +118,17 @@ func newSvcLogOpts(vars svcLogsVars) (*svcLogsOpts, error) {
 			return err
 		}
 		opts.ecs = ecs.New(sess)
-
+		workload, err := configStore.GetWorkload(opts.appName, opts.name)
+		if err != nil {
+			return fmt.Errorf("get workload: %w", err)
+		}
 		newWorkloadLoggerOpts := &logging.NewWorkloadLoggerOpts{
 			App:  opts.appName,
 			Env:  opts.envName,
 			Name: opts.name,
 			Sess: sess,
 		}
-		if opts.targetSvcType != manifest.RequestDrivenWebServiceType {
+		if workload.Type != manifest.RequestDrivenWebServiceType {
```

Comment:
Why do we fetch from SSM instead of using the workspace value? did it result in a bug?

Reply:
See if you like the updated implementation better!

Previously it was introducing a bug because `DeployedWorkload` in `selector` package did not always return `SvcTyp`. This resulted in `opts.targetSvcType` being `""`.